src/FNA3D_Driver_Vulkan.c

```diff
@@ -2241,12 +2241,6 @@ static uint8_t VULKAN_INTERNAL_FindMemoryType(
 		}
 	}
 
-	FNA3D_LogWarn(
-		"Failed to find memory type %X, required %X, ignored %X",
-		typeFilter,
-		requiredProperties,
-		ignoredProperties
-	);
```

Comment:
tbh this warning is the only reason I was able to diagnose this at all - rest looks good but this should really stay.